bin/toV2.ts

```diff
@@ -0,0 +1,46 @@
+type Transformer = (options: TransformerParameters) => { key: string, value: unknown };
+type TransformerParameters = {key: string, value: unknown, manifest: any};
+
+const toV2 = (v3) =>
+	Object.entries(v3)
+		.reduce((v2, [key, value]): Record<string, unknown> => {
+			const transformer = transformers[key] ?? identity;
+			const {key: newKey, value: newValue} = transformer({value, key, manifest: v3});
+			return {...v2, [newKey]: newValue};
+		}, {});
+
+const retainKey = (callback: ({value, manifest}) => unknown): Transformer =>
+	({key, value, manifest}) =>
+		({key, value: callback({value, manifest})});
+
+const fixedValue = (value: unknown) => retainKey(() => value);
+
+const identity = retainKey(({value}) => value);
+const transformers: { [key: string]: Transformer } = {
```

Comment:
this only exists to make our v3 manifest a v2 manifest?

Reply:
Yeah it's used in webpack